modules/optimizing-the-vllm-runtime.adoc

```diff
@@ -1,120 +1,87 @@
-ifdef::upstream[]
-* If you used the pre-installed *vLLM ServingRuntime for KServe* runtime, you duplicated the runtime to create a custom version. For more information about duplicating the pre-installed vLLM runtime, see {odhdocshome}/serving-models/#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
-endif::[]
-ifndef::upstream[]
-* If you used the pre-installed *vLLM ServingRuntime for KServe* runtime, you duplicated the runtime to create a custom version. For more information about duplicating the pre-installed vLLM runtime, see {rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
-endif::[]
 * If you are using the vLLM model-serving runtime for speculative decoding with a draft model, you have stored the original model and the speculative model in the same folder within your S3-compatible object storage.
 
 
 .Procedure
-. From the {productname-short} dashboard, click *Settings* > *Serving runtimes*.
-+
-The *Serving runtimes* page opens and shows the model-serving runtimes that are already installed and enabled.
-. Find the custom vLLM model-serving runtime that you created, click the action menu (&#8942;) next to the runtime and select *Edit*.
-+
-The embedded YAML editor opens and shows the contents of the custom model-serving runtime.
-. To configure the vLLM model-serving runtime for speculative decoding by matching n-grams in the prompt:
-.. Add the following arguments:
+
+. Follow the steps to deploy a model as described in link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#deploying-models-on-the-single-model-serving-platform_serving-large-models[Depyoing models on the single-model serving platform].
+. In the *Serving runtime* field, select the *vLLM ServingRuntime for KServe* runtime.
+. To configure the vLLM model-serving runtime for speculative decoding by matching n-grams in the prompt, add the following arguments under *Additional serving runtime arguments* in the *Configuration parameters* section:
 +
 [source]
 ----
-containers:
-  - args:
-      - --speculative-model=[ngram]
-      - --num-speculative-tokens=<NUM_SPECULATIVE_TOKENS>
-      - --ngram-prompt-lookup-max=<NGRAM_PROMPT_LOOKUP_MAX>
-      - --use-v2-block-manager
+--speculative-model=[ngram]
+--num-speculative-tokens=<NUM_SPECULATIVE_TOKENS>
+--ngram-prompt-lookup-max=<NGRAM_PROMPT_LOOKUP_MAX>
+--use-v2-block-manager
 ----
-.. Replace `<NUM_SPECULATIVE_TOKENS>` and `<NGRAM_PROMPT_LOOKUP_MAX>` with your own values.
+... Replace `<NUM_SPECULATIVE_TOKENS>` and `<NGRAM_PROMPT_LOOKUP_MAX>` with your own values.
```

Comment:
```suggestion
.. Replace `<NUM_SPECULATIVE_TOKENS>` and `<NGRAM_PROMPT_LOOKUP_MAX>` with your own values.
```

Reply:
Ugh, this was a copy paste error. Fixing